### ISSUE TITLE
[BRUKERHISTORIE] Endre navn på feltet "Begrep brukt i datasettet"

### ISSUE THREAD

**Issue by marittor** [Informasjonsforvaltning/behov#357]
Som: datasetteier
Ønsker jeg å: kommunisere tydelig hva "Begrep brukt i datasett" egentlig representerer, nemlig få fram at det er "betydningen av opplysningene i datasettet" som dette gjelder, altså bør overskriften på dette feltet endres
Slik at: vi bruker klarspråk og ikke forvirrer konsumentene.

Relatert til epos:

Lenke til design: 
Lenke til løsningsarkitektur:

Forretningsverdi (Kritisk/Viktig/Mindre viktig/Kjekt å ha):

**Comment by stigbd** [comment]
Fins det eit forslag til kva vi bør endre _til_?

**Comment by RogerSOlsen** [comment]
"Definisjon av sentrale opplysninger i datasett"

**Comment by oystein-asnes** [comment]
Eventuelt korte ned til "Sentrale opplysninger i ... datasettet/tjenesten" Andre forslag @marittor ?

**Comment by oystein-asnes** [comment]
Mi stemme går til "Definisjon av sentrale opplysninger i datasettet". Enig @KurtStian ?

**Comment by KurtStian** [comment]
@oystein-asnes Ja, det høres greit ut. Vi har for tjeneste/hendelse sagt at det er «Begrep som er viktig for å forstå tjenesten/hendelsen» og det blir vel omtrent det samme bare sagt på en annen måte. Så dette kunne jo ha vært et alternativ?

**Comment by KjerstiSteien** [comment]
@oystein-asnes og @KurtStian:

Sånn jeg leser det er det to alternativer her. Hvilket går vi for? 

1: Definisjon av sentrale opplysninger i datasettet
2: Begrep som er viktig for å forstå  datasettet

**Comment by KurtStian** [comment]
> @oystein-asnes og @KurtStian:
> 
> Sånn jeg leser det er det to alternativer her. Hvilket går vi for?
> 
> 1: Definisjon av sentrale opplysninger i datasettet 2: Begrep som er viktig for å forstå datasettet

@KjerstiSteien @oystein-asnes 
Tja, si det. Hvis ser på standardene/spesifikasjonene så sier de omtrent det samme. f.eks.  datasett «...sentrale begrep som er viktige for å forstå og tolke datasettet.»; tjeneste «...begrep som er viktig for å forstå tjenesten.»; modell ...«begrep som er viktig for å forstå eller tolke modellen eller modellelementene.»
Synes 2 er mer klarspråklig/folkelig

**Comment by oystein-asnes** [comment]
Mi stemme går fortsatt til alt 1, (men vi må fleirtalsbøye definisjoner): "Definisjoner av sentrale opplysninger i datasettet".

**Comment by KjerstiSteien** [comment]
Det er en stund siden sist vi diskuterte ordlyden her. 

@KurtStian @Lillebo @kristianeskogvang  Hva er deres innspill her? Synes dere vi bør gå for alternativ 1 eller 2, eller noe helt annet?
